sdcm/cluster.py

```diff
@@ -184,6 +184,20 @@ class Setup(object):
     def test_id(cls):
         return cls._test_id
 
+    @classmethod
```

Comment:
might want to turn this into a property

Reply:
it should be removed once the test_id generated by hydra, will become the document_id in ElasticSearch
think, behavior should be same as for test_id